itests/util/src/main/java/org/apache/hadoop/hive/ql/QTestUtil.java

```diff
@@ -209,14 +211,19 @@ public QTestUtil(QTestArguments testArgs) throws Exception {
     conf.set("test.data.dir", datasetHandler.getDataDir(conf));
     dispatcher.register("dataset", datasetHandler);
     dispatcher.register("replace", replaceHandler);
+    dispatcher.register("scheduledqueryservice", new QTestScheduledQueryServiceProvider());
+    dispatcher.register("scheduledquerycleaner", new QTestScheduledQueryCleaner());
 
     String scriptsDir = getScriptsDir();
 
     this.initScript = scriptsDir + File.separator + testArgs.getInitScript();
     this.cleanupScript = scriptsDir + File.separator + testArgs.getCleanupScript();
 
     postInit();
+    conf.setVar(HiveConf.ConfVars.HIVE_SCHEDULED_QUERIES_EXECUTOR_IDLE_SLEEP_TIME, "1s");
```

Comment:
Default for q tests could be changed by modifying files in `data/conf/*.xml` instead of adding them here (my understanding is that we only to the class those property values that need to be changed at initialization time).